.github/workflows/sync-to-aur.yml

```diff
@@ -0,0 +1,45 @@
+name: Sync package to AUR
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
I'm fairly sure you can write

```
on:
  push: [master]
```

To achieve the same result. But I'm not sure.

Reply:
It does the same.